### ISSUE TITLE
interested in ur project 

### ISSUE THREAD

**Issue by alexgogoing** [peter-mount/global-forecast-system#1]
I lost a package named IOFuction, can u resolve it? will nice to receive ur reply! 
 
![image](https://user-images.githubusercontent.com/30588385/62207058-c1a11780-b3c5-11e9-9990-60502a1102b0.png)

**Comment by peter-mount** [comment]
Yes that's old code but the IOFunction interface is in the [opendata-common](https://github.com/peter-mount/opendata-common) project, specifically [uk/trainwatch/io/IOFunction.java](https://github.com/peter-mount/opendata-common/blob/master/core/src/main/java/uk/trainwatch/io/IOFunction.java)

**Comment by alexgogoing** [comment]
Oh! much thanks！
